Dockerfile

```diff
@@ -13,7 +15,7 @@ ENV VERSION_PKG=github.com/aws/aws-app-mesh-controller-for-k8s/pkg/version
 RUN GIT_VERSION=$(git describe --tags --dirty --always) && \
     GIT_COMMIT=$(git rev-parse HEAD) && \
     BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) && \
-    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build \
```

Comment:
1. i think linux should be replaced with TARGETOS
2. do we need below to work?
```
ARG TARGETOS
ARG TARGETARCH
```

Reply:
I intentionally didn't add these arguments since on Mac (darwin), TARGETOS could be darwin. Let me try to see what manifest gets built and I will update it